.github/workflows/ci.yml

```diff
@@ -12,15 +12,16 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [12.x, 14.x, 16.x, 17.x, 18.x, 19.x]
```

Comment:
Why different versions from `publish.yml`?

Reply:
I have updated the same. `[12.x, 14.x, 16.x, 17.x, 18.x, 19.x]`